programmers/프린터.js

```diff
@@ -0,0 +1,19 @@
+function solution(pri, loc) {
+  const idx = [];
+  let print = 0;
+
+  for (let i = 0; i < pri.length; i += 1) idx.push(i);
+
+  let a;
+  while (true) {
+    if (pri[0] === Math.max.apply(null, pri)) {
+      pri.shift();
+      a = idx.shift();
+      print++;
```

Comment:
```suggestion
      print += 1;
```
no-plusplus
[https://eslint.org/docs/rules/no-plusplus](url)